testsuite/E40.py

```diff
@@ -16,6 +16,20 @@
 
 import foo
 #: Okay
+"docstring"
+
+__version__ == "42"
+
+import foo
+#: Okay
+__author__ == "Simon Gomizelj"
+
+import foo
+#: Okay
+__all_but_with_extra_stuff == "foobar"
```

Comment:
So I'm inclined to think this should be an E402 since it's not explicitly the `__all__`